Example/ConsentViewController_ExampleTests/SPClientCoordinator/SourcePointClient/SourcePointClientSpec.swift

```diff
@@ -329,80 +329,6 @@ class SourcePointClientSpec: QuickSpec {
             }
 
             describe("customConsent") {
```

Comment:
@Nevazhnovu I believe we can delete the entire `describe` block, including that test about the `custom-metrics`. That's already covered in the core-module.